staff.rb

```diff
@@ -0,0 +1,40 @@
+require_relative 'people_in_school'
+
+class Staff < PeopleInSchool
```

Comment:
A `SeniorTeacher` is a `Staff`? I'd go with `StaffMember`, or just `Teacher`.

Also, a staff member/teacher is a `PeopleInSchool`?  Make sure your plurality matches -- Your class should always be named based on what a single instance of that class will represent.  So rather than `PeopleInSchool`, I'd go for `PersonInSchool` or maybe `DBCPerson`